libs/island-ui/core/src/lib/Button/Button.treat.ts

```diff
@@ -200,6 +207,34 @@ const primaryColors: PrimaryColors = (main, text, hover, disabled) => ({
     },
   },
 })
+
+const primaryLightColors: PrimaryLightColors = (
```

Comment:
probably no need to duplicate this function, better to just add disabledText (maybe with default text as color) to the end of the function